packetbeat/protos/udp/udp.go

```diff
@@ -100,7 +100,7 @@ func buildPortsMap(plugins map[protos.Protocol]protos.UDPPlugin) (map[uint16]pro
 	return res, nil
 }
 
-// NewUdp creates and returns a new Udp.
+// NewUDP creates and returns a new Udp.
```

Comment:
```suggestion
// NewUDP creates and returns a new UDP.
```

Reply:
Thanks; if I had re-run gospel that would have been found 😞.